### ISSUE TITLE
Cannot add multiple single line comments

### ISSUE THREAD

**Issue by meaghanlewis** [github/VisualStudio#1162]
__Version__
- GitHub Extension for Visual Studio  version: 2.3.0.20
- Visual Studio version: 2017

__What happened__ 
Go to add a comment on a line in VS. Then try to add a second comment to the same line and you will not be able to. Clicking on the same line will just refresh the comment box.

On .com, try the same thing by adding one comment and then another. You are able to successfully add a second comment to the same line.
<img width="1233" alt="screen shot 2017-08-18 at 11 59 59 am" src="https://user-images.githubusercontent.com/10103121/29473834-24bf00b4-840e-11e7-925a-dbc61935ff86.png">

Going back to VS, after refreshing you are able to see both comments.

**Comment by grokys** [comment]
This would be good to support, though I'm not sure what the UI for it would look like because we place the "Add Comment" and "View Comments" icon in the same place.

**Comment by meaghanlewis** [comment]
Good point. I'm not sure what the UI might look like in this case either.

**Comment by grokys** [comment]
I'm going to remove the `bug` label from this issue, because it's actually by design in that we're not sure what the UI should look like.
